src/main/resources/static/js/knox.js

```diff
@@ -559,7 +559,7 @@ $('#apply-operators-tooltip').click(() => {
           break;
 
         case endpoint.operators.REVERSE:
-          endpoint.designSpaceReverse(inputSpaces.get(0), outputSpace);
+          endpoint.designSpaceReverse(inputSpaces[0], outputSpace);
```

Comment:
Zack, you missed my other comment that this is inconsistent with the other functions. This code should be in the Java file, not here. Please squash these commits when you've fixed them.